code/modules/antagonists/villain/maniac/_maniac.dm

```diff
@@ -155,8 +152,7 @@ GLOBAL_VAR_INIT(maniac_highlander, 0) // THERE CAN ONLY BE ONE!
 		for(var/trait in final_traits)
 			REMOVE_TRAIT(owner.current, trait, "[type]")
 		owner.current.clear_fullscreen("maniac")
-		owner.current.verbs -= /mob/living/carbon/human/proc/toggle_maniac_music
-		owner.current.verbs -= /mob/living/carbon/human/proc/set_custom_music
+		qdel(owner.GetComponent(/datum/component/theme_music))
```

Comment:
Use RemoveComponent instead of qdel them, example:


	var/datum/component/magcom = GetComponent(/datum/component/anti_magic)
	if(magcom)
		magcom.RemoveComponent()

In your case you need to pull it from the owner.